edx_solutions_api_integration/migrations/0002_leaderboard.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.utils.timezone
+from django.conf import settings
+import model_utils.fields
+import opaque_keys.edx.django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('edx_solutions_api_integration', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='LeaderBoard',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, verbose_name='created', editable=False)),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, verbose_name='modified', editable=False)),
+                ('course_key', opaque_keys.edx.django.models.CourseKeyField(max_length=255)),
+                ('position', models.IntegerField()),
+                ('user', models.ForeignKey(to=settings.AUTH_USER_MODEL)),
```

Comment:
@nasirhjafri please re-generate this migration file or edit this to add `on_delete=models.CASCADE` here as well.